server/clients/manageRecallsApiClient.ts

```diff
@@ -303,6 +304,10 @@ export function getServiceMetrics(token: string): Promise<StatisticsSummary> {
   return restClient(token).get<StatisticsSummary>({ path: '/statistics/summary' })
 }
 
+export function getWeeklyRecallsNew(token: string): Promise<GetReportResponse> {
```

Comment:
Can't believe this is me asking you this, but shouldn't you add a PACT?

Reply:
I will look; needs to be added on the API first and isn't there yet.  And given this is almost a 'prototype' feature we may choose to skip it.